lib/subject_importer.rb

```diff
@@ -20,8 +21,6 @@ def run
   attr_reader :path
 
   def targeted_department_numbers
-    @targeted_department_numbers ||= Course.pluck(:number).map do |number|
-      number.split("-").first
-    end.uniq
+    @targeted_department_numbers ||= Department.pluck(:number).map(&:to_s)
```

Comment:
Removed `to_s` on department number and added `to_i` on line 10 -- this will catch any courses with a letter duffix.